src/types/types.ts

```diff
@@ -1,3 +1,19 @@
+export enum Constants{
```

Comment:
https://codingwithroby.substack.com/p/clean-code-the-art-of-clean-naming
https://dev.to/bellonedavide/clean-code-tip-avoid-mental-mappings-55nf

Reply:
https://github.com/PreethiKalyani99/Clockify-Typescript/commit/9e15731dedb28e9b633bd593e355a5ae684cc3c8
https://github.com/PreethiKalyani99/Clockify-Typescript/commit/3f6ae6a87dea0575035d10968920f152900256f6